packages/xo-web/src/common/intl/locales/tr.js

```diff
@@ -1798,7 +1798,7 @@ export default {
   addHostErrorTitle: 'Sunucu ekleme başarısız',
 
   // Original text: "Host patches could not be homogenized."
-  addHostNotHomogeneousErrorMessage: 'Sunucu yamaları homojenleştirilemedi.',
+  addHostsNotHomogeneousErrorMessage: 'Sunucu yamaları homojenleştirilemedi.',
```

Comment:
I think it's better to keep this and to have an incorrect agreement than to have a missing translation :+1: